### PR TITLE
Add rpm2cpio

### PR DESCRIPTION
**Pull request by A6GibKm**

### CODE REVIEW

File: cpio-2.13-mutiple-definition.patch [hunk 1]
```diff
@@ -0,0 +1,605 @@
+
```
Comment (bilelmoussaoui) [inline cpio-2.13-mutiple-definition.patch]: Not sure how this html file is any useful?

Comment (A6GibKm) [reply cpio-2.13-mutiple-definition.patch]: It is not, my bad when "Saving link as".

Comment (A6GibKm) [reply cpio-2.13-mutiple-definition.patch]: For the moment I am not merging this. It seems it is not opening rpm files anyways, at least it does not in my machine.

**Comment by A6GibKm** [comment]
bot, build org.gnome.FileRoller

**Comment by A6GibKm** [comment]
bot, build org.gnome.FileRoller

**Comment by A6GibKm** [comment]
Closing. #8 did the work.

**Pull request closed without merging**